pgjdbc/src/main/java/org/postgresql/util/LruCache.java

```diff
@@ -16,7 +15,7 @@
 /**
  * Caches values in simple least-recently-accessed order.
  */
-public class LruCache<Key extends @NonNull Object, Value extends @NonNull CanEstimateSize>
+public class LruCache<Key extends /* NonNull */ Object, Value extends /* NonNull */ CanEstimateSize>
```

Comment:
In `ArrayEncoding` these were removed, but here commented out. Should we be consistent? I think I prefer them not being present to being commented out, but I do not feel very strongly either way.

Reply:
We should be consistent. I prefer them removed as well